lms/djangoapps/grades/management/commands/backfill_grades.py

```diff
@@ -0,0 +1,141 @@
+            default=100,
+            type=int,
+        )
+        parser.add_argument(
+            '--start_index',
+            help='Offset from which to start processing enrollments.',
+            default=0,
+            type=int,
+        )
+
+    def handle(self, *args, **options):
+        self._set_log_level(options)
+
+        for course_key in self._get_course_keys(options):
+            self.enqueue_backfill_grades_for_course_tasks(course_key, options)
+
+    def enqueue_backfill_grades_for_course_tasks(self, course_key, options):
+        """
+        Enqueue celery tasks to compute and persist all grades for the
+        specified course, in batches.
+        """
+        enrollment_count = CourseEnrollment.objects.num_enrolled_in(course_key)
+        for offset in six.moves.range(options['start_index'], enrollment_count, options['batch_size']):
+            # If any new enrollments are added after the tasks are fired off, they are already persisting grades.
+            # so there is no need to worry about race conditions.
+            task_options = {'routing_key': options['routing_key']} if options.get('routing_key') else {}
+            result = tasks.backfill_grades_for_course.apply_async(
+                kwargs={
+                    'course_key': six.text_type(course_key),
+                    'offset': offset,
+                    'batch_size': options['batch_size'],
+                },
+                options=task_options,
+            )
+            log.info("Created task {task_id} for {course_key} [{offset}...{end}]".format(
+                task_id=result.task_id,
+                course_key=course_key,
+                offset=offset,
+                end=offset + options['batch_size'],
+            ))
+
+    def _get_course_keys(self, options):
+        """
+        Return a list of courses that need scores computed.
+        """
+        courses_mode = get_mutually_exclusive_required_option(options, 'courses', 'all_courses')
+        if courses_mode == 'all_courses':
+            course_keys = [course.id for course in modulestore().get_course_summaries()]
+        else:
+            course_keys = parse_course_keys(options['courses'])
+        self._validate_course_keys(course_keys)
+        return course_keys
+
+    def _validate_course_keys(self, course_keys):
+        """
+        Ensure that course keys are for existent courses.  Raise CommandError
+        for any courses that do not match a real course.
+        """
+        for course_key in course_keys:
+            if not self._course_exists(course_key):
+                raise CommandError("{} refers to a non-existent course".format(course_key))
+
+    def _set_log_level(self, options):
+        """
+        Sets logging levels for this module and the block structure
+        cache module, based on the given the options.
+        """
+        if options.get('verbosity') == 0:
```

Comment:
It is specified in the base, but using it is up to the individual commands.  I think it's left up to individual commands because they don't usually use logging for output, but just do straight print statements.  Logging lets us centralize it a little bit more.